nautobot/core/api/views.py

```diff
@@ -732,16 +732,22 @@ def get(self, request):
         field_name = request.GET.get("field_name")
         app_label, model_name = contenttype.split(".")
         try:
-            model = ContentType.objects.get(app_label=app_label, model=model_name).model_class()
+            model_contenttype = ContentType.objects.get(app_label=app_label, model=model_name)
+            model = model_contenttype.model_class()
+            if model is None:
+                return Response(
+                    f"model for content_type: <{model_contenttype}> not found",
+                    status=400,
```

Comment:
This feels like maybe a 500 error? A valid content-type was requested, but we couldn't find the model class corresponding to it?